events/blocks/events-form/events-form.js

```diff
@@ -495,6 +496,15 @@ async function buildEventform(bp, formData) {
   }
 }
 
+function eventFormSendAnalytics(bp,view) {
+  if (bp.modal) {
+    const title = bp?.title?.textContent?.trim();
+    const name = title ? ` | ${title}` : '';
+    const modalId = bp?.modal?.id ? ` | ${bp?.modal?.id}` : '';
```

Comment:
You don't need to ? check for modal again here since you ruled it out in the if-statement.